.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,14 @@
-<!-- Learn the overall process and best practices for pull requests at https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/repository-management.md#pull-requests. -->
+<!--
 
-<!-- Gutenberg's license is in the process of updating to be dual-licensed under the GPL and MPL. As part of that transition, all new contributions are dual-licensed. For more information, see: https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md -->
+Hi there! Thanks for contributing to Gutenberg!
+
+If this is your first time contributing, you may find reviewing these guides first to be helpful:
+- Overall process and best practices for pull requests: https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/repository-management.md#pull-requests.
+- Coding Standards: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/
+- Accessibility testing: https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/accessibility-testing.md
+- Gutenberg licensing: https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md
```

Comment:
🤔 As a follow up, it might be interesting to have a bot add the link as a comment for first time contributors. If there's a lot of text in a template I suspect most folks will tend to skim the content.

Reply:
> I personally feel there's too much blabla here that would encourage folks to just ignore the rest 

I personally agree, too, it still feels too verbose even after reducing the number of links. We should probably have a single link to a very simple index page containing these links so that the PR template remains simple and documentation is not overwhelming.

Reply:
We already have it covered so we might need only to edit the comment:

https://github.com/WordPress/gutenberg/blob/82d87ff38c000e877ce5df11771730291898d47a/packages/project-management-automation/lib/tasks/first-time-contributor-label/index.js#L61-L71

Reply:
I've reduced the instructions to a minimum by linking to the Contributor Guidelines and adding any other relevant links to them.

Reply:
In case it's necessary we know what work on next 👍🏻